tests/platform/windows/windows-kurzweil-testSpec.txt

```diff
@@ -0,0 +1,10 @@
+windows-kurzweil-testSpec.js
+
+Descriptions:
+This will run the acceptance tests for Kurzweil 3000 (KESI). It uses 2 NP sets: kurzweil_common.json and kurzweil_application.json
+
+Prerequisites:
+* Kurzweil 3000 needs to be installed
+* Need to set up username "gpii" for Kurzweil(incomplete)
```

Comment:
Hi,
(incomplete) means that we haven't come to a conclusion on how to handle the issue.

If we can get the user's Kurzweil username from the user, I believe we can locate the user's Kurzweil settings folder and change the settings.